packages/dashmate/src/docker/DockerCompose.js

```diff
@@ -378,21 +378,15 @@ class DockerCompose {
     }
 
     // Check docker compose
-    if (process.env.DOCKER_COMPOSE_V2) {
-      try {
-        await execAsync('docker compose');
-      } catch (e) {
-        throw new Error('Docker Compose V2 is not installed');
-      }
-    } else {
-      if (!hasbin.sync('docker-compose')) {
-        throw new Error('Docker Compose is not installed');
-      }
+    try {
+      await execAsync('docker compose');
```

Comment:
I would recommend you do both checks in one call. I mean, calling `await dockerCompose.version()` must be enough to check binaries and version?